_posts/en/pages/2016-01-15-lifecycle.md

```diff
@@ -53,9 +53,23 @@ Once EOL is reached, you will need to upgrade to a newer version.
 
 _TBA: to be announced_
 
-## Foot notes
+## Protocol versioning
```

Comment:
Ok.  I'll add another short paragraph to the bottom of this section stating that.